src/sig_stateful/sig_stfl.h

```diff
@@ -61,7 +61,78 @@ extern "C" {
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h10_w2 "10/2"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h10_w4 "10/4"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h10_w8 "10/8"
+
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h15_w1 "15/1"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h15_w2 "15/2"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h15_w4 "15/4"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h15_w8 "15/8"
+
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h20_w1 "20/1"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h20_w2 "20/2"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h20_w4 "20/4"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h20_w8 "20/8"
+
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h25_w1 "25/1"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h25_w2 "25/2"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h25_w4 "25/4"
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h25_w8 "25/8"
+
+/* LM registry */
+#define OQS_SIG_STFL_alg_lm_sha256_n32_h5  "LM-SHA256_N32_H5"  //0x00000005
+#define OQS_SIG_STFL_alg_lm_sha256_n32_h10 "LM-SHA256_N32_H10" //0x00000006
+#define OQS_SIG_STFL_alg_lm_sha256_n32_h15 "LM-SHA256_N32_H15" //0x00000007
+#define OQS_SIG_STFL_alg_lm_sha256_n32_h20 "LM-SHA256_N32_H20" //0x00000008
+#define OQS_SIG_STFL_alg_lm_sha256_n32_h25 "LM-SHA256_N32_H25" //0x00000009
+
+/* LM-OTS registry */
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w1 "LMOTS-SHA256_N32_W1" //0x00000001
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w2 "LMOTS-SHA256_N32_W2" //0x00000002
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w3 "LMOTS-SHA256_N32_W4" //0x00000003
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w4 "LMOTS-SHA256_N32_W8" //0x00000004
+
+/* Defined LM parameter sets */
+/* Algorithm identifier for LMS-SHA256_N32_H5 */
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h5 "LMS-SHA256_N32_H5"   //0x00000005
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h10 "LMS-SHA256_N32_H10" //0x00000006
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h15 "LMS-SHA256_N32_H15" //0x00000007
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h20 "LMS-SHA256_N32_H20" //0x00000008
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lms_sha256_n32_h25 "LMS-SHA256_N32_H25" //0x00000009
+
+/* LM-OTS registry */
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w1 "LMOTS-SHA256_N32_W1" //0x00000001
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w2 "LMOTS-SHA256_N32_W2" //0x00000002
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w3 "LMOTS-SHA256_N32_W4" //0x00000003
+
+/* Algorithm identifier for XMSSMT-SHA2_60/12_256 */
+#define OQS_SIG_STFL_alg_lmots_sha256_n32_w4 "LMOTS-SHA256_N32_W8" //0x00000004
 
 /**
  * Returns identifiers for available signature schemes in liboqs.  Used with OQS_SIG_STFL_new.
```

Comment:
Suppose comment should read "...for available stateful signature schemes..."? Or should this code fit identically into the current `liboqs` sig APIs?

Reply:
Will fix these comments.